proto/viam/app/build/v1/build.proto

```diff
@@ -83,3 +99,79 @@ message ListJobsResponse {
   // jobs is ordered by (build start time, alphabetical platform).
   repeated JobInfo jobs = 1;
 }
+
+message RepoLink {
+  // viam internal ID which maps to an external oauth app
+  string oauth_app_link_id = 1;
+  // OrgID of the module
+  string org_id = 2;
+  // public namespace of the module
+  optional string namespace = 3;
+  // ID of the module (just the name, not the dotted org:module form)
+  string module_id = 4;
+  // git repo in owner/repository form
+  string repo = 5;
```

Comment:
```suggestion
  // github repo in owner/repository form
  string repo = 5;
```
?

Reply:
felt uncomfortable using a vendor name in our api; and also this could hypothetically be used in other places in the future